.github/CODEOWNERS

```diff
@@ -1,7 +1,2 @@
-# Example Github CODEOWNERS file
-# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners for more information.
-# This is a comment.
-# Each line is a file pattern followed by one or more owners.
-#
-# In general, CODEOWNERS should be team in the Cray-HPE organization, not individuals.
-#
+# Hardware Management Team Members and Administrators
+*  @Cray-HPE/hardware-management @Cray-HPE/hardware-management-admin
```

Comment:
@mjendrysik-hpe I assume you are the admin of this repo? Can you add the  `@Cray-HPE/hardware-management` group with maintain permissions, and `@Cray-HPE/hardware-management-admin` with admin permissions?

Reply:
Done